src/Sylius/Bundle/ApiBundle/QueryHandler/GetCustomerStatisticsHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\QueryHandler;
```

Comment:
```suggestion
namespace Sylius\Bundle\ApiBundle\Admin\QueryHandler;
```
What do you think about organizing the admin context commands/queries into the sub-namespace?

Reply:
Despite we use it in the admin section now, there's no problem with using it to provide statistics for the customer, so the same query can be used in both `admin` and `shop` sections. IMO there's no need for it now.

Reply:
```suggestion
namespace Sylius\Bundle\ApiBundle\QueryHandler\Admin;
```

If sub-namespace must be used, then it should be in  `QueryHandler` instead before `QueryHandler`